flake.nix

```diff
@@ -58,6 +58,12 @@
       url = "github:phadej/HTTP";
       flake = false;
     };
+    iserv-proxy = {
+      type = "git";
+      url = "https://gitlab.haskell.org/ghc/iserv-proxy.git";
+      rev = "6e95df7be6dd29680f983db07a057fc2f34f81f6";
+      flake = false;
+    };
```

Comment:
Yikes, this is going to suck, as we need to potentially keep this in sync with GHC. Or I guess more likely ensure it works across multiple GHC versions.

Reply:
Yes I was hoping that we could make it work for multiple ghc versions with minimal changes and perhaps a small amount of CPP.  Might also make sense to put it in hackage instead of using git.